x-pack/solutions/observability/plugins/apm/test/scout/ui/fixtures/page_objects/transaction_details.ts

```diff
@@ -7,10 +7,16 @@
 
 import type { KibanaUrl, ScoutPage } from '@kbn/scout-oblt';
 import { waitForApmSettingsHeaderLink } from '../page_helpers';
+import { BIGGER_TIMEOUT } from '../constants';
 
 export class TransactionDetailsPage {
   constructor(private readonly page: ScoutPage, private readonly kbnUrl: KibanaUrl) {}
 
+  // Locators
+  public get waterfallSizeWarning() {
```

Comment:
NIT: this can be defined as a locator, like: 
```
  public apmWaterfallSizeWarning: Locator;

  constrictor .... 

    this.serviceMap = page.testSubj.locator('apmWaterfallSizeWarning');


```